### PR TITLE
feat(s3-service)!: move to AWS-SDK v3

### PR DESCRIPTION
**Pull request by AndrewBarba**
This should most likely be released as a breaking version change

### CODE REVIEW

File: lib/s3-service.js [hunk 1]
```diff
@@ -7,7 +7,39 @@
...
+exports.getSignedUrl = async (
+  type,
+  { Expires, ...params },
+  callback = () => {}
+) => {
+  let command;
+  switch (type) {
+    case 'getObject':
+      command = new GetObjectCommand(params);
+      break;
```
Comment (naorpeled) [inline lib/s3-service.js:33]: What about putObject?

**Review by naorpeled — COMMENTED** [review]
Overall looks good,
left a comment, once that's addressed and CI passes, we can merge this :)

File: lib/s3-service.js (hunk 1 above)
Comment (AndrewBarba) [reply lib/s3-service.js:33]: Am I missing something, where is putObject used in this library?

Comment (naorpeled) [reply lib/s3-service.js:33]: my bad, you're right.
Let's just make sure CI passes before we merge this in

**Comment by ygsnyder** [comment]
Any movement on this fix?

**Comment by naorpeled** [comment]
> Any movement on this fix?

I'll get it merged tomorrow hopefully

**Pull request merged**